### ISSUE TITLE
Repository browser freeze when browsing mirrored repository

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#214]
```
What steps will reproduce the problem?
1. Open Repository panel
2. Add http://svn.alfresco.com/repos/alfresco-open-mirror/alfresco/HEAD/root as 
a repository, give it a name
3. Open the created repository to browse history

What is the expected output? What do you see instead?
The open repository browser starts listing the change history. At some point it 
will stop and svnx freezes and have to be restarted.
After how many listed changes this happens depends recent activity. See also 
below under additional info.

What version of the product are you using? On what operating system?
1.3.4

Please provide any additional information below.
The Alfresco public repository is the public available mirror of their gigantic 
internal subversion repository. Their internal repo includes non-public source, 
and this is filtered out in the public mirror. This I think means that there 
are null/empy commits, revision numbers appear in list but contain no info. 
Svnx might hang on those. I've had no problem checking out source from this 
mirror, it is just the direct browsing of hte repository that is the problem.

```

Original issue reported on code.google.com by `pe...@loftux.se` on 2 Oct 2014 at 5:08

**Comment by GoogleCodeExporter** [comment]
```
The problem is that some (several hundred) log entries contain no date which 
causes svnX to throw an exception when it tries to display them.
However, svnX downloads the log & creates a valid log cache file.

You can work around this by quitting svnX, opening 
~/Library/Caches/com.lachoseinteractive.svnX/A00000?????.cache in TextWrangler 
& doing a global search & replace of `<key>date</key>\r\t\t\t<string></string>` 
for `<key>date</key> <string>2014-11-01T00:00:00</string>`.  Save the file & 
relaunch svnX.
[The actual cache file name varies but should be the most recently modified & 
about 30MB.]
```

Original comment by `chris...@gmail.com` on 1 Nov 2014 at 3:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svnX 2.0b1.
```

Original comment by `chris...@gmail.com` on 24 Aug 2015 at 7:05
- Changed state: **Fixed**
